content/techniques/performance.md

```diff
@@ -74,6 +74,38 @@ You can pass options into the Fastify constructor through the `FastifyAdapter` c
 new FastifyAdapter({ logger: true });
 ```
 
+
+#### Middleware
+
+Middleware functions (when `@nestjs/platform-fastify` is used) retrieve the raw `req` and `res` objects instead of Fastify's wrappers. This is how the `middie` package works (that's used under the hood) and `fastify` - check out this [page](https://www.fastify.io/docs/latest/Reference/Middleware/) for more information,
```

Comment:
```suggestion
Middleware functions retrieve the raw `req` and `res` objects instead of Fastify's wrappers. This is how the `middie` package works (that's used under the hood) and `fastify` - check out this [page](https://www.fastify.io/docs/latest/Reference/Middleware/) for more information,
```